src/main/java/core/basesyntax/handlers/ReturnOperationHandler.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.handlers;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidDataException;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class ReturnOperationHandler implements OperationHandler {
+    @Override
+    public void calculateOperation(FruitTransaction transaction) {
+        if (transaction.getQuantity() < 0) {
+            throw new InvalidDataException("Negative quantity");
+        } else if (transaction.getFruit() == null) {
+            throw new IllegalArgumentException("Invalid fruit type");
+        }
```

Comment:
I guess it will be better. The same in other handlers
```suggestion
        if (transaction.getQuantity() < 0) {
            throw new InvalidDataException("Negative quantity");
        } 
        if (transaction.getFruit() == null) {
            throw new IllegalArgumentException("Invalid fruit type");
        }
```